### PR TITLE
Check in/65597/v3 buttons

### PR DESCRIPTION
**Pull request by brianseek**
## Summary

Updates buttons to use the USWDS v3 DS va-button component. Buttons in test files were not updated. The va-button-pair was not used due to translation and button text issues. But DS tickets were created. See this PR's ticket for more details.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#65597

## Testing done

- _Describe what the old behavior was prior to the change_

Updates unit and e2e tests.

## What areas of the site does it impact?

Day-of and pre-check-in

## Acceptance criteria
- [ ] All buttons rendering in the user facing app use the USWDS v3 DS component 

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

**Review by CBonade — DISMISSED** [review]

**Review by wullaski — CHANGES_REQUESTED** [review]
I only found one small issue, besides the back button issue where you go back to answer yes to the travel review question after choosing file later.

File: src/applications/check-in/components/AppointmentDisplay/CheckInButton.jsx [hunk 1]
```diff
@@ -34,28 +34,35 @@ const CheckInButton = ({
     },
     [checkInWindowEnd, eventRecorder, getCurrentPageFromRouter, onClick],
   );
-
+  if (isCheckingIn) {
+    // Creating an new button because even having disabled=false results in a SR showing the button as dimmmed.
+    return (
+      <va-button
```
[the remark this answers is not in the document]
Comment (brianseek) [reply src/applications/check-in/components/AppointmentDisplay/CheckInButton.jsx:40]: nice find!

**Review by wullaski — DISMISSED** [review]
LGTM! 🚢

**Review by wullaski — DISMISSED** [review]

File: src/applications/check-in/components/AppointmentDisplay/CheckInButton.jsx [hunk 2]
```diff
@@ -34,28 +34,38 @@ const CheckInButton = ({
     },
     [checkInWindowEnd, eventRecorder, getCurrentPageFromRouter, onClick],
   );
-
+  if (isCheckingIn) {
```
Comment (lee-delarm6) [inline src/applications/check-in/components/AppointmentDisplay/CheckInButton.jsx:37]: Do you want to keep the comment here?

Comment (brianseek) [reply src/applications/check-in/components/AppointmentDisplay/CheckInButton.jsx:37]: yes, just a little documentation for why there are two button renders.

**Review by lee-delarm6 — APPROVED** [review]
Looks good 👍

**Pull request merged**